models/Notification.php

```diff
@@ -32,12 +32,12 @@ class Notification extends AbstractModel
     protected $id;
 
     /**
-     * @var int
+     * @var string
```

Comment:
why can a date be a string?

Reply:
It was string, I fixed docblock only. If you want I can change on DateTime object,

Reply:
hmm, in DB it is a timestamp therefore an INT, isn't it? 
so we should change all typehints to INT or - since it is not main focus of this PR - just leave it as it was